### DISCUSSION TITLE
Window corner rounded

### DISCUSSION THREAD

**Discussion by shootheart** [soreau/pixdecor#24] [Q&A]
Hi,
Excuse me for not understanding it accurately, but I found it seems just border rounded.
Is there a way to make actual window corner rounded?

**Comment by soreau** [comment]
Yes, there is. However, you risk cutting off parts of the window that matter. See [this](https://wayfire.org/2020/10/28/Writing-Plugins.html) for more information. Currently, this is not planned for pixdecor, so closing for now.
